.eslintrc.js

```diff
@@ -0,0 +1,69 @@
+// @ts-check
+
+/** @type { import('eslint').Linter.Config } */
+const config = {
+  extends: [
+    'eslint:recommended', // https://eslint.org/docs/rules/
+    'plugin:react/recommended', // https://github.com/jsx-eslint/eslint-plugin-react#list-of-supported-rules
+    'plugin:@typescript-eslint/recommended', // https://typescript-eslint.io/rules/
+    'react-app', // https://github.com/facebook/create-react-app/blob/main/packages/eslint-config-react-app/index.js
+    'plugin:prettier/recommended', // See .prettierrc
+  ],
+  parser: '@typescript-eslint/parser',
+
+  settings: {
+    // Warn about <a target="_blank" rel="noopener"> in components other than "a"
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md#custom-link-components
+    linkComponents: ['Chip', 'Link', 'SocialLink', 'StyledLink'],
```

Comment:
This sucks in a few ways.

- It does not follow renames 
  ```tsx
  import { Link as RouterLink } from 'react-router-dom'
  ```

- 'Link' detects
  ```tsx
  import Link from '@mui/material/Link'
  <Link href="https://aaa.com" target="_blank"></Link>
  ```
  but not 
  ```tsx
  import { Link } from 'react-router-dom'
  <Link to="https://aaa.com" target="_blank"></Link>
  ```

- `{ name: 'Link', linkAttribute: 'to' }, { name: 'Link', linkAttribute: 'href' }` ignores the first

Reply:
Indeed, this rule doesn't seem to be that useful. I am for simply dropping it. What do you thing @lukaw3d ?

Reply:
Especially since, according to [the docs](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md#when-to-override-it), 

> Modern browsers (Chrome ≥ 88, Edge ≥ 88, Firefox ≥ 79 and Safari ≥ 12.2) automatically imply rel="noopener". Therefore **this rule is no longer needed**, if legacy browsers are not supported. 

So I would say that we can simply drop this, can't we?